papaparse.js

```diff
@@ -426,8 +426,8 @@
 	function ChunkStreamer(config)
 	{
 		this._handle = null;
-		this._paused = false;
 		this._finished = false;
+		this._complete = false;
```

Comment:
If this variable is used to indicate that complete has been already called I prefer to name it `completed`

Reply:
Done.